config_defaults.yml

```diff
@@ -574,19 +574,19 @@ Gemfile:
           - mingw
           - x64_mingw
       - gem: 'puppet-module-posix-default-r#{minor_version}'
-        version: '~> 0.3'
+        version: '~> 0.4'
```

Comment:
Do we need to worry about 0.4.0 ? as that broke some stuff.

Reply:
The situation with 0.4.0 does not change under this change, as `~> 0.3` already matches `'0.4.0'`:
```
david@davids:~$ pry
[1] pry(main)> d = Gem::Dependency.new('test', '~> 0.3')
=> Gem::Dependency.new("test", Gem::Requirement.new(["~> 0.3"]), :runtime)
[2] pry(main)> d.match? 'test', '0.4.0'
=> true
[3] pry(main)> 
```